source/components/Login/LoginContainer.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+import Login from './Login'
+
+export default class LoginContainer extends Component {
+  static redirect(path) {
+    location.href = path
+  }
+
+  render() {
+    return <Login redirect={ LoginContainer.redirect } />
```

Comment:
This does not appear to be working. Undefined is being passed to the child, not the intended redirect function.